packages/client/src/__tests__/integration/happy/blog-env-cockroachdb/schema.prisma

```diff
@@ -0,0 +1,36 @@
+datasource db {
+  provider = "cockroachdb"
+  url      = env("TEST_COCKROACH_URI")
+}
+
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["Cockroachdb"]
```

Comment:
```suggestion
  previewFeatures = ["cockroachdb"]
```
unless intentional

Reply:
Casing doesn't matter for previewFeatures though